embeddings/pipeline/lightning_sequence_labeling.py

```diff
@@ -38,10 +38,16 @@ def __init__(
         accelerator: Optional[Union[str, Accelerator]] = "auto",
         predict_subset: LightingDataModuleSubset = LightingDataModuleSubset.TEST,
         load_dataset_kwargs: Optional[Dict[str, Any]] = None,
+        model_checkpoint_kwargs: Optional[Dict[str, Any]] = None,
     ):
         task_train_kwargs = config.task_train_kwargs
         task_train_kwargs.update({"devices": devices, "accelerator": accelerator})
         tokenizer_name_or_path = tokenizer_name_or_path or embedding_name_or_path
+        model_checkpoint_kwargs = (
+            model_checkpoint_kwargs
+            if model_checkpoint_kwargs
+            else LightningPipeline.DEFAULT_MODEL_CHECKPOINT_KWARGS
+        )
```

Comment:
if user passes kwargs (e.g. `{"save_last": True}`), all of the default kwargs are overwritten. Is it intentional?

Reply:
Moreover if the passed dict is empty i.e.  `{}`, it is overwritten by default kwargs.

Reply:
Fixed

Reply:
Do you use `DEFAULT_MODEL_CHECKPOINT_KWARGS` now?

Reply:
Currently it's the same behaviour as with other `DEFAULT_KWARGS` User can use it, but automatically is not set.

Reply:
removed